code/modules/atmospherics/components/omni_devices/mixer.dm

```diff
@@ -109,6 +109,8 @@
 	var/transfer_moles_max = INFINITY
 
 	for (var/datum/omni_port/P in inputs)
+		if(!P.concentration)
+			continue
 		transfer_moles += (set_flow_rate*P.concentration/P.air.volume)*P.air.total_moles
 		transfer_moles_max = min(transfer_moles_max, calculate_transfer_moles(P.air, output.air, delta, (output && output.network && output.network.volume) ? output.network.volume : 0))
```

Comment:
I believe it makes more sense to sum in this line: the min seems just wrong.

```transfer_moles_max = 0
for(blah)
  transfer_moles_max += calculate_transfer_moles(P.air, output.air, delta, (output && output.network && output.network.volume) ? output.network.volume : 0)
```

To be perfectly honest the rest of the logic here seems wrong as well; the `P.concentration` in the line above, for example, should likely be omitted, and this computation is in general not precise.